onnx/backend/test/case/node/castlike.py

```diff
@@ -87,14 +87,14 @@ def export() -> None:
                     helper.tensor_dtype_to_np_dtype(getattr(TensorProto, from_type))
                 )
                 if "STRING" == to_type:
-                    # Converting input to str, then give it np.object dtype for generating script
+                    # Converting input to str, then give it np.object_ dtype for generating script
                     ss = []
                     for i in input.flatten():
                         s = str(i).encode("utf-8")
                         su = s.decode("utf-8")
                         ss.append(su)
 
-                    output = np.array(ss).astype(np.object).reshape([3, 4])
+                    output = np.array(ss).astype(np.object_).reshape([3, 4])
```

Comment:
Is `object` better than `np.object_` here?

> To give a clear guideline for the vast majority of cases, for the types bool, object, str (and unicode) using the plain version is shorter and clear, and generally a good replacement.

https://numpy.org/devdocs/release/1.20.0-notes.html#using-the-aliases-of-builtin-types-like-np-int-is-deprecated

Reply:
+1 I think object is better, which is just like what this PR: https://github.com/onnx/onnx/pull/3306 did.